File: 0455-assign-cookies/0455-assign-cookies.py
class Solution(object):
    def findContentChildren(self, g, s):
        g.sort()
        s.sort()
        n=len(g)
        m=len(s)
        e=0
        f=0
        while e<n and f<m:
            if(g[e]<=s[f]):
                e+=1
            f+=1
        return e
        """
        :type g: List[int]
        :type s: List[int]
        :rtype: int
        """
        

File: 0455-assign-cookies/0455-assign-cookies.cpp
class Solution {
public:
    int findContentChildren(vector<int>& g, vector<int>& s) {
        sort(g.begin(),g.end());
        sort(begin(s),end(s));
        int n=g.size();
        int m=s.size();
        int e=0;
        for(int f=0;e<n && f<m;f++){
            if(g[e]<=s[f]) e++;
        }
        return e;
    }
};